### PR TITLE
I added node instructions

### PR DESCRIPTION
**Pull request by Ozzy4201**

### CODE REVIEW

**Comment by Soldacool** [comment]
So i am very incompetent and it does that when i run the script 
 ##
image

**Comment by Soldacool** [comment]
Aight nvm i figured out im just dumb

**Comment by ASOwnerYT** [comment]
it assumes u use windows :( :( :( :( :( :(

**Comment by User9684** [comment]
sir you could've just used node's website telling them that "if you're localhosting, install node from their website, run `npm clone https://github.com/User9684/stupid-discord-badge` and then run `npm run run tokenhere` :p

File: readme.md [hunk 1]
```diff
@@ -12,9 +12,23 @@
...
 - Once you are eligible, get the badge at the [Application site](https://discord.com/developers/active-developer)
 
+## How to use on your very own computer wow wow
+- Install [Node.js](https://nodejs.org/en/)
+- Create an application at the [Developer panel](https://discord.com/developers/applications/) (or you can use a previously existing one)
+- Make it into a bot if it isn't already one
+- Obtain your token
+- If you have git installed, clone this repo, otherwise download the zip file from the funny little green code button
+- Extract to a folder
+- Type "cmd" in the address bar at the top of your windows file explorer to open a console, and type "npm run run BOT_TOKEN_HERE"
```
Comment (KTibow) [inline readme.md:22]: are we just assuming noobs use windows

Comment (Ozzy4201) [reply readme.md:22]: Yes, as it is the most used OS in the world.

**Pull request closed without merging**